diff --git a/node_modules/rn-sliding-up-panel/SlidingUpPanel.js b/node_modules/rn-sliding-up-panel/SlidingUpPanel.js
index 09bb2d1..9ecd5c2 100644
--- a/node_modules/rn-sliding-up-panel/SlidingUpPanel.js
+++ b/node_modules/rn-sliding-up-panel/SlidingUpPanel.js
@@ -3,7 +3,6 @@ import PropTypes from 'prop-types'
 import clamp from 'clamp'
 
 import {
-  ViewPropTypes,
   UIManager,
   TextInput,
   Keyboard,
@@ -56,8 +55,8 @@ class SlidingUpPanel extends React.PureComponent {
     showBackdrop: PropTypes.bool,
     backdropOpacity: PropTypes.number,
     friction: PropTypes.number,
-    containerStyle: ViewPropTypes.style,
-    backdropStyle: ViewPropTypes.style,
+    containerStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+    backdropStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
     children: PropTypes.oneOfType([PropTypes.element, PropTypes.func])
   }
 
